test/hotspot/jtreg/compiler/c2/TestBitCompressValueTransform.java

```diff
@@ -26,7 +26,7 @@
  * @bug 8350896
  * @library /test/lib /
  * @summary C2: wrong result: Integer/Long.compress gets wrong type from CompressBitsNode::Value.
- * @run main/othervm -Xbatch -XX:-TieredCompilation -XX:CompileThresholdScaling=0.3 compiler.c2.TestBitCompressValueTransform
+ * @run main/othervm compiler.c2.TestBitCompressValueTransform
```

Comment:
Now that you have no flags any more, I think we can use the `driver`.
```suggestion
 * @run driver compiler.c2.TestBitCompressValueTransform
```